content/github-staff/github-shared-components.mdx

```diff
@@ -21,3 +24,5 @@ Please note that these components are **owned by the feature teams**, and will n
 
 - [Handling new patterns](/guides/contribute/handling-new-patterns)
 - [Upstreaming to Primer](/guides/contribute/adding-new-components#upstreaming-to-primer)
+
+## Shared components
```

Comment:
Now that we have this section, I think the "Finding these components" section is redundant. Let's move the content from that section into the new one.